go.mod

```diff
@@ -22,8 +22,8 @@ require (
 	github.com/elazarl/go-bindata-assetfs v0.0.0-20160803192304-e1a2a7ec64b0
 	github.com/envoyproxy/go-control-plane v0.9.5
 	github.com/frankban/quicktest v1.11.0 // indirect
-	github.com/gogo/protobuf v1.2.2-0.20190723190241-65acae22fc9d
-	github.com/golang/protobuf v1.3.5
+	github.com/gogo/protobuf v1.3.2
+	github.com/golang/protobuf v1.4.3
```

Comment:
Yeah we can't do this right now if it involves updating `golang/protobuf` to 1.4+. It runs into the `gogo/protobuf` + `golang/protobuf` incompatibility that we have yet to resolve.

Reply:
oh okay :(
I thought it was an easy fix, it imply a lot of changes